tests/test_telegramobject.py

```diff
@@ -269,7 +291,7 @@ def test_pickle_apply_api_kwargs(self):
         assert obj.foo == "bar"
         assert obj.api_kwargs == {}
 
-    async def test_pickle_removed_and_added_attribute(self):
+    async def test_pickle_backwards_compatibility(self):
```

Comment:
I mean its technically forwards compatibility isn't it? Older pickle files should work with newer versions.

Reply:
https://en.wikipedia.org/wiki/Backward_compatibility

> A complementary concept is [forward compatibility](https://en.wikipedia.org/wiki/Forward_compatibility). A design that is forward-compatible usually has a [roadmap](https://en.wikipedia.org/wiki/Technology_roadmap) for compatibility with future standards and products.[[2]](https://en.wikipedia.org/wiki/Backward_compatibility#cite_note-Zeldman-2)

We didn't really design old pickle files to be compatible with this change though 🤔